### ISSUE TITLE
src build from source broken due to inconshreveable/go-update/check import

### ISSUE THREAD

**Issue by peggyl** [sourcegraph/srclib#201]
Import from https://github.com/sourcegraph/srclib/blob/master/src/selfupdate_cmd.go#L8 fails:

`github.com/inconshreveable/go-update/check` 

because of this commit: https://github.com/inconshreveable/go-update/commit/1d9fbfa25a6e94ff049e2ec9e48ea4e30d5e2856

Repro: `go get -u -v sourcegraph.com/sourcegraph/srclib/cmd/src` (from https://srclib.org/gettingstarted/#building-from-source) without v1.0 checkout of github.com/inconshreveable/go-update in $GOPATH.

**Comment by beyang** [comment]
Thanks for reporting!

Is this on `master` branch? The latest development branch is actually `nodb`, which we hope to merge sometime this week, so if you don't mind waiting for that, that's probably the best path forward.

**Comment by stephenwithav** [comment]
It's in `master`.  I just ran into the same problem.

I downloaded the binary as a temporary solution.

**Comment by beyang** [comment]
Yep, build from the `nodb` branch. That is ahead of master significantly. The version of `go-update` is 68f5725818189545231c1fd8694793d45f2fc529.
